README.md

```diff
@@ -1,10 +1,10 @@
-# JavaScript Module Fragments
+# JavaScript Module Declarations
 
-Champion: Daniel Ehrenberg (Igalia, in partnership with [eyeo](https://eyeo.com/))
```

Comment:
We should be careful to disclose the sponsorship somewhere or other. It has now been partly eyeo and partly Bloomberg:)